packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -141,7 +142,8 @@ const MOCK_NETWORK = {
   provider: MAINNET_PROVIDER,
   blockTracker: { getLatestBlock: () => '0x102833C' },
   state: {
-    network: '5',
+    networkId: '5' as NetworkId,
```

Comment:
Yeah, okay, I thought that might happen as well.

One quick idea is to export `assertNetworkId` from NetworkController and use it. That said, we already know it's a network ID and TypeScript should be able to see that it is, so that seems a bit silly.

Would it be worth it to define a `MockNetwork` type like this:

```
type MockNetwork = {
  provider: ProviderProxy;
  blockTracker: BlockTrackerProxy;
  state: NetworkState;
  subscribe: (listener: (state: NetworkState) => void) => void;
};
```

and then assign it to `MOCK_NETWORK`, `MOCK_MAINNET_NETWORK`, and `MOCK_CUSTOM_NETWORK`? (You could work out how to give a type to `MOCK_NETWORK_WITHOUT_CHAIN_ID` too, but that's a special case.)

Reply:
Using a mock network type seemed to work: https://github.com/MetaMask/core/pull/1196/commits/94e34f37b8be4ed6cfa9cf030215947dd2b606a7